app/assets/stylesheets/earthworks.css

```diff
@@ -152,3 +152,10 @@ Uncomment lines 142-144
     display: none;
   }
 }
+
+.expand-collapse .btn:disabled {
```

Comment:
I don't think you need the `:disabled` pseudo selector here.

Reply:
I am not sure I understand your meaning. I want these styles to show up only when the button is disabled.

Reply:
They will. You set variables for disabled styles.

Reply:
Gotcha. That makes sense.